### PR TITLE
docs: Synchronizing top level content with website

### PR DESCRIPTION
**Pull request by eddie-knight**
- Navbar now populates from included content directories
- Markdown files in included directories now populate as sub-pages within that section
- "Edit this on Github" now links auto-populated files to the correct corresponding location

[See preview site to validate changes.](https://deploy-preview-1249--tag-security.netlify.app)

### CODE REVIEW

File: assessments/projects/nats/self-assessment.md
Comment (eddie-knight) [inline assessments/projects/nats/self-assessment.md:281]: The build fails without this change now that all assessments are being brought in as sub-pages

**Comment by eddie-knight** [comment]
Linters are crying because of the self-assessment I had to touch to get the build working 🤣 Not sure whether it's worse practice to add a full lint of that otherwise-unrelated file to this review, or to ignore the linters.

**Comment by eddie-knight** [comment]
split off the linting so that it doesn't crowd this PR.

blocked by #1252

**Comment by eddie-knight** [comment]
Checks are all passing now.

Before we merge, I've requested input from @brandtkeller regarding the approach here

File: website/Makefile
Comment (brandtkeller) [inline website/Makefile:14]: What do you think about checking for existing front-matter? Establishes an opt-in process for adjusting titles and other content.

Comment (eddie-knight) [reply website/Makefile:14]: Seems like it would be a good improvement. Might be suited for a follow-up PR?

File: website/Makefile
Comment (brandtkeller) [inline website/Makefile:29]: The only problem I see here currently is that anytime we reference a `README.md` that was translated to an `_index.md` that link is now broken. 

examples from the homepage:
- governance/governance process link
- related-groups link

This may be more widespread throughout the website - IE if you click on the `governance` navbar option and then try `related-groups` you'll encounter an error. yet using the sidebar option for `related groups` works as expected.

File: website/Makefile
Comment (brandtkeller) [reply website/Makefile:14]: Titles sometimes are duplicated - curious if we could check for existence and if not otherwise present - replace a `h1` with the same title?

Comment (eddie-knight) [reply website/Makefile:14]: Good catch- glad you're doing a more thorough look through. I should have time on Friday to look for a fix unless you beat me to it.

File: website/Makefile
Comment (brandtkeller) [reply website/Makefile:29]: Will do some experimenting as I have time to think about work-arounds.

Comment (brandtkeller) [reply website/Makefile:29]: One work-around - although duplicative - would be to copy both the `README.md` (thus retaining any references to the file) as well as copying the contents of the `README.md` into the `_index.md` file. 

Current `include-markdown` option is breaking links even to direct children. IE from the navbar -> governance -> try to click on any-one of the links - all are broken even though they are present and accessible from the sidebar.

Comment (eddie-knight) [reply website/Makefile:29]: I think this is sorted with the latest change. Let me know what you think.

**Comment by eddie-knight** [comment]
Just made a push incorporating Option#2 above

**Review by PushkarJ — APPROVED** [review]
Thank you for working on this!

@brandtkeller once you feel this is good enough for now, feel free to approve it :) for future incremental changes we can open an issue so we don't lose track of them in PR comments.  

Really appreciate how far we have come with website refresh in last few weeks, thanks to you @brandtkeller 

Thank you Eddie for continuing the momentum on this!

File: website/Makefile
Comment (brandtkeller) [reply website/Makefile:29]: agreed! Looks resolved.

File: website/Makefile
Comment (brandtkeller) [reply website/Makefile:14]: This is still an outlier - but maybe a feature we can add in a separate issue.

**Review by brandtkeller — APPROVED** [review]
Tested a bunch of workflows - all are working much better after the recent chanegs. This really improves the website for the TAG quite considerably and sets the stage for a variety of content to be produced. 

@PushkarJ - @eddie-knight did all the work here - I just jumped into the fray to help.

**Pull request merged**
